db/migrate/20150122204353_create_password_resets.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePasswordResets < ActiveRecord::Migration
+  def change
+    create_table :password_resets do |t|
+      t.integer :user_id, null: false
+      t.string :token, limit: 128, null: false
```

Comment:
http://blog.jonanin.com/2013/11/20/postgresql-char-varchar/

This forces a VARCHAR where it could instead be TEXT. +1 to removing the limit.